res/css/views/rooms/_ThreadSummary.pcss

```diff
@@ -55,8 +55,8 @@ limitations under the License.
         bottom: 0;
         width: 60px;
         box-sizing: border-box;
-        /* XXX: We use `$system-transparent` instead of `transparent` to work around a Safari <15.4 bug */
-        background: linear-gradient(270deg, $system 50%, $system-transparent 100%);
+        /* XXX: We use an HEXA `transparent` to work around a Safari <15.4 bug */
+        background: linear-gradient(270deg, $system 50%, #e1e6ec00 100%);
```

Comment:
Looks like this will be no longer flexible for custom themes

Reply:
I don't think i have a better solution for this `#e1e6ec00` this would require a color function in any case to be backwards compatible. 

This will do a transition to a hardcoded transparent hex code, which seems good enough given the name of the variable 🤔

Do you see alternatives here?

Reply:
Could we not keep the `$system-transparent` variable and set it to `#e1e6ec00` in the light base theme so all themes inherit, and let `_custom` override it with `var(--system-transparent)`